dev-qt/qtcore/qtcore-5.14.9999.ebuild

```diff
@@ -44,6 +44,16 @@ QT5_GENTOO_PRIVATE_CONFIG=(
 
 PATCHES=( "${FILESDIR}/${PN}-5.14.1-cmake-macro-backward-compat.patch" ) # bug 703306
 
+pkg_pretend() {
+	if use kernel_linux; then
```

Comment:
`use kernel_linux || return` to reduce indentation in the remainder of the function